projects/Mallard/src/screens/settings/gdpr-consent-screen.tsx

```diff
@@ -79,7 +79,8 @@ const GdprConsent = ({
             settings.gdprAllowFunctionality &&
             settings.gdprAllowFunctionality
```

Comment:
I know this wasn't you but is this right or should we have`gdprAllowPerformance` here?

Reply:
Fixed.